src/plugins/controls/public/react_controls/controls/types.ts

```diff
@@ -40,15 +40,15 @@ export type DefaultControlApi = PublishesDataLoading &
   HasType &
   HasUniqueId &
   HasParentApi<ControlGroupApi> & {
-    // Can not use HasSerializableState interface
-    // HasSerializableState types serializeState as function returning 'MaybePromise'
-    // Controls serializeState is sync
-    serializeState: () => SerializedPanelState<DefaultControlState>;
-    /** TODO: Make these non-public as part of https://github.com/elastic/kibana/issues/174961 */
```

Comment:
I decided there is no reason to lock down the API at this point in time, especially since we are no longer allowing external plugins to register their own control types. It honestly felt like overcomplicating to split this into "public" and "private" APIs 😅 So I just removed this `TODO`